modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb

```diff
@@ -68,7 +68,7 @@ def upstream_healthcheck
     private
 
     def upstream_service_details(service_name, health_checker, time)
-      healthy = health_checker.send("#{service_name.snakecase}_is_healthy?")
+      healthy = health_checker.healthy_service?(service_name.snakecase)
```

Comment:
I'd drop the `snakecase` here since the regex in `healthy_service?` matches ignoring case and the service_names passed in are based on `AppealsApi::HealthChecker::SERVICES` I'd assume they wouldn't need to be massaged anyway

Reply:
@erluti ready to merge? Thanks for your review.